dev-support/ranger-docker/Dockerfile.ranger-hbase

```diff
@@ -22,14 +22,15 @@ ARG HBASE_PLUGIN_VERSION
 
 COPY ./dist/version                                             /home/ranger/dist/
 COPY ./dist/ranger-${HBASE_PLUGIN_VERSION}-hbase-plugin.tar.gz  /home/ranger/dist/
-COPY ./downloads/hbase-${HBASE_VERSION}-bin.tar.gz              /home/ranger/dist/
 
-COPY ./scripts/ranger-hbase-setup.sh                     /home/ranger/scripts/
-COPY ./scripts/ranger-hbase.sh                           /home/ranger/scripts/
-COPY ./scripts/ranger-hbase-plugin-install.properties    /home/ranger/scripts/
-COPY ./scripts/hbase-site.xml                            /home/ranger/scripts/
+COPY ./scripts/ranger-hbase-setup.sh                            /home/ranger/scripts/
+COPY ./scripts/ranger-hbase.sh                                  /home/ranger/scripts/
+COPY ./scripts/ranger-hbase-plugin-install.properties           /home/ranger/scripts/
+COPY ./scripts/hbase-site.xml                                   /home/ranger/scripts/
 
-RUN tar xvfz /home/ranger/dist/hbase-${HBASE_VERSION}-bin.tar.gz --directory=/opt/ && \
+RUN wget -O /home/ranger/dist/hbase-${HBASE_VERSION}-bin.tar.gz \
```

Comment:
files like hbase-<version>-bin.tar.gz could be large. It will help to avoid downloading such files for every build. Please review.